diff -urN pyorbit-2.24.0/configure.in pyorbit-2.24.0.new/configure.in
--- pyorbit-2.24.0/configure.in	2008-09-23 14:27:43.000000000 +0200
+++ pyorbit-2.24.0.new/configure.in	2008-12-29 19:28:20.000000000 +0100
@@ -33,6 +33,9 @@
 AC_DISABLE_STATIC
 AC_PROG_LIBTOOL
 
+dnl when using libtool 2.x create libtool early, because it's used in configure
+m4_ifdef([LT_OUTPUT], [LT_OUTPUT])
+
 AM_PATH_PYTHON([2.4])
 
 AM_CHECK_PYTHON_HEADERS(,[AC_MSG_ERROR(could not find Python headers)])
